### ISSUE TITLE
Scroll bar is hidden behind menu when large number of results are returned

### ISSUE THREAD

**Issue by TurtleTech8** [DenisLabrecque/Bible-Browser#59]

**Comment by DenisLabrecque** [comment]
Is this the top of the scrollbar, or is this something else?
![receptus scrollbar corner](https://user-images.githubusercontent.com/11930465/72348695-932ba600-36a0-11ea-8d0d-bdca27c68cf6.PNG)

**Comment by TurtleTech8** [comment]
Yeah, it's the top the scroll bar there. It is sitting underneath the three dots so when the bar is small enough, it hides underneath the dots.
